net/cgmnlm/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           legacysupport 1.1
+
+# getline, fmemopen
+legacysupport.newest_darwin_requires_legacy 16
+
+name                cgmnlm
+version             1.6
+revision            0
+categories          net gemini
+license             GPL-3
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         ${name} is a colorful Gemini line mode client
+long_description    {*}${description}
+
+homepage            https://git.sr.ht/~rwa/cgmnlm
```

Comment:
I think we have a `sourcehut` portgroup.

Reply:
Ok, I can try to use that in a while.